tkg/client/cluster.go

```diff
@@ -180,7 +180,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			// If `features.cluster.auto-apply-generated-clusterclass-based-configuration` feature-flag is not activated
 			// log command to use to create cluster using ClusterClass based config file and return
 			if !c.IsFeatureActivated(constants.FeatureFlagAutoApplyGeneratedClusterClassBasedConfiguration) {
-				log.Warningf("\nTo create a cluster with it, use")
+				log.Warningf("\nTo create a cluster with it, please use the following command together with all customized flags set in the last step. e.g. --tkr, -v")
```

Comment:
@hongtaiwang may I know why the feature gate `auto-apply-generated-clusterclass-based-configuration` must to to be enabled when create a cluster with a kubernetes-style yaml file? 
As far as I know, when create a classy workload-cluster with k8s-style yaml file, code logic will not go here. It should be go into here: https://github.com/vmware-tanzu/tanzu-framework/blob/main/tkg/client/cluster.go#L147-L156. So enable this feature gate with a k8s-style yaml file may not make sense.

Also the error is reported by the user is more like a mapping error for `VSPHERE_TEMPLATE` var. At the first step, user specifies `--tkr v1.23.16---vmware.1-tkg.2`, but why the value of this var is parsed to be `/TSV_DC/vm/ubuntu_temp_v1.23.16`? What's the correct template name for v1.23.16---vmware.1-tkg.2 tkr?

BTW, the `-v` is used for debugging AFAIK, so may it's not necessary. Or does it do something else?

Reply:
better to have a zoom call to carify this:)